src/main/webapp/app/guided-tour/guided-tour.service.ts

```diff
@@ -208,6 +206,14 @@ export class GuidedTourService {
         this.resetTour();
     }
 
+    public checkTourStateFinished(guidedTour: GuidedTour): boolean {
+        const tourSetting = this.guidedTourSettings.filter(setting => setting.guidedTourKey === guidedTour.settingsKey);
+        if (tourSetting.length > 0 && tourSetting[0].guidedTourState.toString() === GuidedTourState[GuidedTourState.FINISHED]) {
```

Comment:
It would be less error prone and remove the .toString call if the enum was using string identifiers:
```
export enum GuidedTourState {
    STARTED = 'STARTED',
    FINISHED = 'FINISHED',
}
```

Reply:
@thilo-behnke This change would cause changes for the database as well. 
@krusche If you want to have the Strings stored in the database, I can do the implementation in another PR.